src/compute/compute_mesh.cpp

```diff
@@ -293,10 +322,13 @@ specfem::compute::mesh::mesh(
   this->ngllx = this->quadratures.gll.N;
   this->ngllz = this->quadratures.gll.N;
 
-  this->points = this->assemble();
+  this->points = this->assemble(adjacency_map, this->mapping);
```

Comment:
Why do you pass `adjacency_map` and `mapping` as parameters when they can be directly accessed by `this->assemble`